templates/staff/report_publish_request_detail.html

```diff
@@ -0,0 +1,97 @@
+{% extends "staff/base.html" %}
```

Comment:
I'm not sure that this needs to be a separate page. Could it be a card on the Report staff detail page instead? I appreciate there will be more than a single button needed at some point, but I think Tom O could help figure out how to make that work. Similarly, on the list view, the publish request could be a filter option, so that someone could view all outstanding (or cancelled) requests.

I also don't think that the Analysis Request staff detail page should have the Publish Request on it either. 

It might be helpful to think of this in terms of an aggregate root, in DDD speak (https://martinfowler.com/bliki/DDD_Aggregate.html).

Reply:
#3108 deals with several of these points!

It adds a second button (reject), wrapping both of the buttons up so we can have a disabled state with hover to explain.

It switches the two publish request models to have FKs to their relevant target model so it's possible (and I think likely with reports) that we'll have N publish requests for each report.

It adds more states to the reports list page's filter-by-published-state now.

I'd be happy to look at consolidating these pages once all the core publishing features are in and we've got some user feedback but lets keep them around while we're developing it.  They're really useful during dev atm!